src/StatsdClient/StatsdClient.nuspec

```diff
@@ -1,16 +1,20 @@
-<?xml version="1.0"?>
-<package >
-  <metadata>
-    <id>DogStatsD-CSharp-Client</id>
-    <version>2.2.1</version>
-    <authors>Datadog</authors>
-    <owners>Datadog</owners>
-    <licenseUrl>https://github.com/DataDog/dogstatsd-csharp-client/blob/master/MIT-LICENCE.md</licenseUrl>
-    <projectUrl>https://github.com/DataDog/dogstatsd-csharp-client</projectUrl>
-    <iconUrl>https://si0.twimg.com/profile_images/1380708189/dd_logo2_icon.png</iconUrl>
-    <requireLicenseAcceptance>false</requireLicenseAcceptance>
-    <description>A DogStatsD client for C#. DogStatsD is an extension of the StatsD metric server for use with Datadog. For more information visit http://datadoghq.com.</description>
-    <copyright>Copyright 2016</copyright>
-    <tags>statsd dogstatsd datadog</tags>
-  </metadata>
-</package>
+<?xml version="1.0"?>
+<package>
+  <metadata>
+    <id>DogStatsD-CSharp-Client</id>
+    <version>2.2.1</version>
+    <authors>Datadog,Nearmap</authors>
+    <owners>Datadog,Nearmap</owners>
+    <title>DogStatsD-CSharp-Client</title>
+    <requireLicenseAcceptance>false</requireLicenseAcceptance>
+    <licenseUrl>https://github.com/DataDog/dogstatsd-csharp-client/blob/master/MIT-LICENCE.md</licenseUrl>
+    <description>A DogStatsD client for C# (v2.2.1). Forked by Nearmap to adjust the assembly name and root namespace to avoid conflict with statsd. DogStatsD is an extension of the StatsD metric server for use with Datadog. For more information visit http://datadoghq.com.</description>
+    <projectUrl>https://github.com/nearmap/dogstatsd-csharp-client</projectUrl>
+    <iconUrl>https://si0.twimg.com/profile_images/1380708189/dd_logo2_icon.png</iconUrl>
+    <copyright>Copyright &#169; 2018</copyright>
+    <tags>statsd dogstatsd datadog nearmap</tags>
+  </metadata>
```

Comment:
this should probably have different `id`, `version` since Klondike is a local cache for 3rd-party packages & so it probably already has the Datadog package cached.

Reply:
Yeah, it's somewhat hidden at the moment.

Reply:
I've decided to leave the version the same, so we can easily match with DataDog's version. I imagine that at most we'll only ever have one Nearmap package per DogStatsdClient package.